.github/workflows/releasebot.yml

```
@@ -0,0 +1,248 @@
+        id: message
+        uses: actions/github-script@v7
+        env:
+          RELEASES: ${{ inputs.PublishedPackages }}
+          PACKAGE_MAP: ${{ steps.package-map.outputs.result }}
+          BASE_URL: ${{ steps.base-url.outputs.result }}
+        with:
+          result-encoding: string
+          script: |
+            const baseUrl = new URL(process.env.BASE_URL);
+            const releases = JSON.parse(process.env.RELEASES || '[]');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)